scripts/auto-rebase/assets.yaml

```diff
@@ -45,12 +45,7 @@ assets:
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_role.yaml
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_rolebinding.yaml
-      - file: topolvm-controller_rbac.authorization.k8s.io_v1_role.yaml
```

Comment:
In #1481 I had to update the lvms component to not try to load these assets. You're also removing more files than I removed. Were some of these already unused?

Reply:
I looked into this again, some of lvms manifests in our repo had names changed and it broke the rebase

Reply:
Actually, I might just bring them back and let rebase job do its job so we get nice changelog in PR's description